src/app/components/UserMenu/index.tsx

```diff
@@ -43,91 +43,140 @@ export default function UserMenu() {
-        <MenuIcon className="h-6 w-6" />
-      </Menu.Button>
-      <Menu.List position="left">
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("discover");
-          }}
-        >
-          <GlobeIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("discover")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("publishers");
-          }}
-        >
-          <TransactionsIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("websites")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            navigate("/send");
-          }}
-        >
-          <SendIcon className="w-6 h-6 -ml-0.5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.send")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            navigate("/receive");
-          }}
-        >
-          <ReceiveIcon className="w-6 h-6 -ml-0.5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.receive")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("transactions");
-          }}
-        >
-          <TransactionsIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("transactions")}
-        </Menu.ItemButton>
-        <Menu.Divider />
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("settings");
-          }}
-        >
-          <GearIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("settings")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            openOptions("accounts");
-          }}
-        >
-          <AddressBookIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("accounts")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            utils.openUrl("https://feedback.getalby.com");
-          }}
-        >
-          <RocketIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("feedback")}
-        </Menu.ItemButton>
-        <Menu.ItemButton
-          onClick={() => {
-            utils.openUrl("https://guides.getalby.com");
-          }}
-        >
-          <QuestionIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("help")}
-        </Menu.ItemButton>
-        <Menu.Divider />
-        <Menu.ItemButton onClick={lock}>
-          <LockIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
-          {tCommon("actions.lock")}
-        </Menu.ItemButton>
-      </Menu.List>
-    </Menu>
-  );
+  if (window.location.pathname == "/options.html") {
```

Comment:
This can be simplified like below:

```js
// ...
<Menu.List position="left">
  {window.location.pathname !== "/options.html" && (
     <>
       <Menu.ItemButton
         onClick={() => {
           openOptions("discover");
         }}
       >
         <RocketIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
         {tCommon("discover")}
       </Menu.ItemButton>
       // ...till divider
     </>
  }
  // ...settings, accounts till lock
</Menu.List>

Reply:
yeah, didn't noticed that previosly. Will fix it now.

Reply:
yeah, didn't noticed that previosly. Will fix it now.

Reply:
fixed !!